services/blockchain-connector/app.js

```diff
@@ -26,6 +26,7 @@ const nodeStatus = require('./shared/nodeStatus');
 
 const { getGenesisBlock } = require('./shared/sdk/genesisBlock');
 const { getSchemas } = require('./shared/sdk/endpoints_1');
+const { init } = require('./shared/sdk/network');
```

Comment:
Kindly add an `index.js` file under `./shared/sdk` and add all the init logic there.

```suggestion
const { init } = require('./shared/sdk');
```